packages/provider-elasticsearch/src/index.js

```diff
@@ -37,7 +37,21 @@ let ElasticsearchProvider = (config = { request: {} }) => ({
       },
     }
   },
-  async runSearch({ requestOptions = {} } = {}, node, schema, filters, aggs) {
+  async runSearch(
+    { requestOptions = {} } = {},
+    node,
+    schema,
+    filterData,
+    aggData
+  ) {
+    //Unpack filters and query data if hoisted data present, or use regular values
+    let aggs = aggData && aggData.hoistProps ? aggData.aggs : aggData
+    let filters =
```

Comment:
This is still the same kind of stuff we were doing before with omit. Lets not complicate things like this if we can avoid it.
In fact, do we even need to hoist to the root? The prior implementation hoisted to a parent agg level which @rudolph9 thought might be more efficient

Reply:
This came about due to when you select intersections and click show results, the meta filters encapsulate the filters from the report node on its run. In this case the filters are brought in without an. So in this way if a hoisted property is needed it has to be above all of the filters for it to work properly.

I will look into just adding the hoisting without the unpacking in results function, but I think the filters were the problematic part here when running on a result node from a report node selection.

Reply:
If we need to hoist to the root, that's fine - but juggling like this seems like incidental complexity. Ideally we can just call something to get hoisted values directly, and not sometimes pack them up in filters/aggs and conditionally unpack.

Reply:
I should be able to add a step in the server project, but that would add to the documented flow so it seemed like a bigger impact than just getting it working in the pivot case first. 

I can rewrite the server to attachHoisted props and pass to the curriedSearch like relevant Filters I just need to work out if I could do the same with the aggs as well which I think I could with minimal impact, but this will also have a bigger impact over the whole code so I was trying to avoid that. Maybe we can meet up for a little to talk about it?

Reply:
Also the getHoistedProps does get them directly now from the groups with no need to pass anything around, it really just comes down to needing to persist them when relevant Filters include the aggregation nodes as the source of the filters in the server package.

Reply:
I addressed this issue in the latest PR, please let me know your thoughts.